docs/software/social.md

```diff
@@ -1,4 +1,9 @@
-# Social / Search
+---
+title: Social and Publishing Software Packages
+description: Description of DietPi software options related to social platforms and publishing servers
+---
+
+# Social & Publishing
```

Comment:
Let's stick with "Social & Search" in this PR, otherwise we need to change it in the navigation menu and in the `dietpi-software` script as well.

But you are right, after YaCy moved to distributed projects, "Search" is wrong. Let's however discuss in a dedicated issue how to call the category instead.

Reply:
Ok. Changed.

Reply:
Opened an issue about that, so we do not forget: #5298